src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,46 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkUserLogin(user.getLogin());
+        checkUserPassword(user.getPassword());
+        checkUserAge(user.getAge());
+        storageDao.add(user);
+        return user;
```

Comment:
```suggestion
        return storageDao.add(user);
```

Reply:
I think this methods:
        checkUserLogin();
        checkUserPassword();
        checkUserAge();
should be private

Reply:
I agree with you. It was my mistake)